integration/200_clustering_2_test.sh

```diff
@@ -2,24 +2,30 @@
 
 . ./config.sh
 
-start_suite "Launch 2 scopes and check they cluster"
-
-weave_on $HOST2 launch
-weave_on $HOST2 launch-dns
-docker_on $HOST2 run -dit --name db1 peterbourgon/tns-db
-container_id=$(docker_on $HOST2 run -dit --name app1 --link db1:db1 peterbourgon/tns-app)
-
-scope_on $HOST1 launch $HOST2
-scope_on $HOST2 launch $HOST1
-
-SUCCESS=
-for i in {1..10}; do
-  if (curl -s $HOST1:4040/api/topology/containers | grep "$container_id" >/dev/null); then
-    SUCCESS=1
-    break
-  fi
-  sleep 1
-done
-assert "echo $SUCCESS" "1"
-
-end_suite
+start_suite "Launch 2 scopes and check they cluster automatically"
+
+weave_on $HOST1 launch $HOST1 $HOST2
+weave_on $HOST2 launch $HOST1 $HOST2
+
+sleep 10 # let weave settle
```

Comment:
Now thats weave pushed the DNS polling to 10s it might not be necessary.  Let me try.